how-it-works/5_governance/00_tokenomics.card.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Tokenomics
+---
+
+![](/img/how-it-works/direct-integration-with-bitcoin.600x300.jpg)
+
+# Tokenomics
+
+The Internet Computer (IC) is underpinned by advanced tokenomics. 
+It makes use of an utility token called ICP, which is used for a variety of functions on the platform including participation in governance and paying for computation & storage. 
+In addition, the IC's tokenomic model provides a funding framework for exciting Web3 projects being built on the Internet Computer. 
+
+[Go deeper](/how-it-works/tokenomics/)
```

Comment:
link existiert nicht

Reply:
Thanks @sesi200, I have created a new page on tokenomics see here (https://github.com/dfinity/portal/pull/696/files). 

I thought I could link to it via "[Go deeper](/how-it-works/tokenomics/)" as we use a similar link, namely "[Go deeper](/how-it-works/network-nervous-system-nns/)" in this card https://raw.githubusercontent.com/dfinity/portal/master/how-it-works/5_governance/01_network-nervous-system-nns.card.md.

Can you advice?

Reply:
Ah, then we'll just have to merge #696 before

Reply:
Yes, and also this file upload. https://github.com/dfinity/portal/pull/695/files

For the next time: Should I have bundled this ?

Reply:
Yes, it makes sense to do changes that depend on each other in the same PR. Easier to merge :)